### PR TITLE
tiproxy: add experimental warning for v0.2.0

### PR DESCRIPTION
**Pull request by djshow832**
### What is changed, added or deleted? (Required)

Add a warning for tiproxy v0.2.0.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiproxy/tiproxy-overview.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ title: TiProxy 简介
 summary: 介绍 TiProxy 的主要功能、安装与使用方法。
 ---
 
+> **警告：**
+>
+> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tiproxy/issues) 反馈。
```
Comment (Oreoxmt) [inline tiproxy/tiproxy-overview.md:8]:
```suggestion
> TiProxy 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tiproxy/issues) 反馈。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tiproxy/tiproxy-overview.md [hunk 2]
```diff
@@ -3,6 +3,10 @@ title: TiProxy 简介
...
+
 # TiProxy 简介
```
Comment (Oreoxmt) [inline tiproxy/tiproxy-overview.md:11]:
```suggestion

# TiProxy 简介

> **警告：**
>
> TiProxy 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tiproxy/issues) 反馈。

```

**Review by hfxsd — APPROVED** [review]

File: tiproxy/tiproxy-overview.md [hunk 3]
```diff
@@ -5,6 +5,9 @@ summary: 介绍 TiProxy 的主要功能、安装与使用方法。
 
 # TiProxy 简介
 
+> **警告：**
+>
+> TiProxy 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tiproxy/issues) 反馈。
 TiProxy 是 PingCAP 的官方代理组件，它放置在客户端和 TiDB server 之间，为 TiDB 提供负载均衡、连接保持、服务发现等功能。
```
Comment (Oreoxmt) [inline tiproxy/tiproxy-overview.md:11]:
```suggestion
TiProxy 是 PingCAP 的官方代理组件，它放置在客户端和 TiDB server 之间，为 TiDB 提供负载均衡、连接保持、服务发现等功能。

```

File: tiproxy/tiproxy-overview.md [hunk 4]
```diff
@@ -5,8 +5,12 @@ summary: 介绍 TiProxy 的主要功能、安装与使用方法。
...
 
+
```
Comment (Oreoxmt) [inline tiproxy/tiproxy-overview.md:13]:
```suggestion

TiProxy 是 PingCAP 的官方代理组件，它放置在客户端和 TiDB server 之间，为 TiDB 提供负载均衡、连接保持、服务发现等功能。

```

**Pull request merged**